tests/functions.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+require '../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class Functions extends TestCase
+{
+    public function testPrepareOutput_success1()
+    {
+        $result = prepareOutput([['id' => 'idtest', 'book_title' => 'title', 'author' => 'author', 'genre' => 'genre', 'page_count' => 'pages', 'year_released' => 'year']]);
+        $this->assertisArray($result);
+    }
+
+    public function testPrepareOutput_success2()
+    {
+        $result = prepareOutput([['id' => 'idtest', 'book_title' => 'title', 'author' => 'author', 'genre' => 'genre', 'page_count' => 'pages', 'year_released' => 'year']]);
+        $expected = [['<pre><span class="class"><h2>title</h2><p><em>author</em></p><br>Released in year. genre. pages pages.</span></pre>']];
+        $this->assertEquals($result, $expected[0]);
+    }
```

Comment:
You could probs actually combine these two tests. You could run both assertions in the same test and still get the same output result.